ADAL/src/ADAuthenticationError.m

```diff
@@ -96,7 +96,7 @@ - (id)initInternalWithDomain:(NSString *)domain
     {
         NSString* codeStr = [self getStringForErrorCode:code domain:domain];
         AD_LOG_ERROR(correlationId, @"Error raised: (Domain: \"%@\" Code: %@ ProtocolCode: \"%@\"", domain, codeStr, protocolCode);
-        AD_LOG_ERROR_PII(correlationId, @"Error details: %@", details);
+        AD_LOG_ERROR_PII(correlationId, @"Error raised: (Domain: \"%@\" Code: %@ ProtocolCode: \"%@\". Error details: %@", domain, codeStr, protocolCode, details);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 163 characters long line (which is greater than 120 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=objc%3ALineLength)